bifrost/app/blog/blogs/llm-stack-guide/src.mdx

```diff
@@ -1,77 +1,114 @@
+
+| Service         | Tools                                                                                                                  |
+| :-------------- | :------------------------------------------------------------------------------------------------------------------------------ |
 | Datastores      | [Helicone API](https://docs.helicone.ai/rest/request/post-v1requestquery), LangFuse, LangSmith                                  |
 | Experimentation | [Helicone Experiments](https://docs.helicone.ai/use-cases/experiments#how-to-run-llm-prompt-experiments), Promptfoo, Braintrust |
 | Evaluators      | PromptFoo, Lastmile, BaseRun                                                                                                    |
 | Fine Tuning     | OpenPipe, Autonomi                                                                                                              |
 
-# Helicone: Your Gateway to LLM Excellence
 
-Helicone isn't just another tool in the LLM stack - it's a game-changer. Our primary focus areas, Gateway and Observability, are crucial for building robust, scalable LLM applications.
+## Where Helicone Fit Into Your LLM Stack
+
+Helicone isn't just another tool in the LLM stack - it's a game-changer for building AI applications. Our primary focus areas, Gateway and Observability, are crucial for building robust, scalable LLM applications. We offer:
+
+1. **Efficient LLM Cache**: Optimize your API calls and reduce costs.
+2. **Real-Time Monitoring**: Track your embedding models and external API usage.
+3. **Orchestration Framework**: Streamline your data processing and machine learning workflows.
+
+![Helicone in the emerging LLM Stack](/static/blog/llm-stack-guide/production.webp)
+
+## The Future of LLM Applications
 
-![Helicone in the LLM Stack](/static/pasted_images/llm-stack-helicone-in-picture.png)
+As we look ahead, several trends are shaping the future of LLM applications:
 
-By integrating Helicone into your LLM workflow, you're not just adopting a tool; you're embracing a philosophy of efficiency, scalability, and deep insights into your LLM applications.
+- Increased focus on specialized, domain-specific models
+- Enhanced privacy and security measures for LLM deployments
+- Improved integration between LLMs and traditional software stacks
+- Advancements in multimodal AI capabilities
 
-Ready to supercharge your LLM development? Dive deeper into Helicone's capabilities and see how it can transform your LLM stack today!
+The LLM Stack is revolutionizing AI application development, enabling more robust and scalable solutions. As LLM architectures evolve, the focus is on creating cost-effective, powerful AI systems that expand the frontiers of machine learning and artificial intelligence. This new paradigm empowers developers to meet the growing demands of our AI-driven world with increased efficiency and innovation.
```

Comment:
I'd just remove "The Future of LLM Applications" section because doesn't really have any good substance